docs/gettingstarted.md

```diff
@@ -0,0 +1,99 @@
+<!--[metadata]>
++++
+title = "Getting Started"
+description = "Getting started with Notary"
+keywords = ["docker, notary, trust, image, signing, collection, cli"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Getting Started
+
+On this page you create a trusted collection of data using Notary, add data, and publish it to a remote Notary server.
+
+## Prerequisites
+
+Make sure you have already
+[installed Notary](install.md) and are running a [Notary Server](notary-server.md).
+
+## Step 1: Initialization
+
+In order to be able to add content to a trusted collection, you first need to initialize it.
+
+```
+$ notary init example.com/collection
```

Comment:
This GUN is slightly different from the one we use in `basic-operations.md`.  This doesn't really matter for purposes of demoing commands in either guide for now, but we may want to uniformly use `user` in the example collection in the future to make delegations easier to understand?